chromadb/db/clickhouse.py

```diff
@@ -220,14 +220,14 @@ def delete_collection(self, name: str):
         """
         )
 
+        self._delete_index(collection_uuid)
```

Comment:
Should we add a python test for delete collection? I assume this was error'ing before?

Reply:
Yes, I'd like to land that after this if that is ok

Reply:
I've added a test now.